examples/using-nerv/server.js

```diff
@@ -1,29 +0,0 @@
-const port = parseInt(process.env.PORT, 10) || 3000
-const dev = process.env.NODE_ENV !== 'production'
-const moduleAlias = require('module-alias')
-
-// For the development version, we'll use React.
-// Because, it support react hot loading and so on.
-if (!dev) {
-  moduleAlias.addAlias('react', 'nervjs')
```

Comment:
By removing this react is used for SSR and nerv for client side rendering. Probably not what you'd want.

Reply:
Indeed! That could get confusing.

Thanks for the update.